.github/workflows/test_windows_job.yml

```diff
@@ -27,12 +27,12 @@ jobs:
       runner: windows.8xlarge.nvidia.gpu
       test-infra-repository: ${{ github.repository }}
       test-infra-ref: ${{ github.ref }}
-      job-name: "win-py3.8-cu116"
+      job-name: "win-py3.8-cu117"
       timeout: 60
       script: |
         conda create --yes --quiet -n test python=3.8
         conda activate test
-        python -m pip install --extra-index-url https://download.pytorch.org/whl/nightly/cu116 --pre torch
+        python -m pip install ---index-url=https://download.pytorch.org/whl/nightly/cu117 --pre torch
```

Comment:
Problem here was that the CUDA 11.6 index no longer has any `torch` binaries. Since we additionally used `--extra-index-url`, we pulled the wheel from PyPI: https://github.com/pytorch/test-infra/actions/runs/4883695947/jobs/8715408340?pr=4104#step:9:86

And the Windows wheel on PyPI is not built against CUDA.

This is not related to this PR other than this PR triggered this test.